src/components/AddMovie.jsx

```diff
@@ -1 +1,63 @@
-// implement AddMovie component here
+import React from 'react';
+import PropTypes from 'prop-types';
+import InputTitle from './forms/inputTitle';
+import InputSubtitle from './forms/inputSubtitle';
+import InputImage from './forms/inputImage';
+import InputTextArea from './forms/inputTextArea';
+import InputNumber from './forms/inputNumber';
+import Select from './forms/select';
+import Button from './forms/button';
+
+class AddMovie extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      subtitle: '',
+      title: '',
+      imagePath: '',
+      storyline: '',
+      rating: 0,
+      genre: 'action',
+    };
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
```

Comment:
Acredito que ficaria mais visível assim.
```suggestion
const { name, value } = event.target;
    this.setState({
      [name]: value,
    });
```